modules/theme/src/themes/default/collections/grid.overrides

```diff
@@ -366,6 +366,21 @@
     }
 }
 
+/*----------------------------------
+     TOTP
+------------------------------------*/
+
+.ui.grid {
+    &.totp {
```

Comment:
Since this is only applicable to `My Account` let's remove the grid variation and add the styles in `user-portal.less`.

**Suggestion**

Use a `div` to add the CSS class. 

```
<div className="totp">
```